garage/sampler/base.py

```diff
@@ -1,195 +1,39 @@
-from dowel import logger, tabular
-import numpy as np
+from abc import ABC, abstractmethod
```

Comment:
please only import packages from stdlib. this make it easier to spot where dependencies come form.

e.g.

```python
import abc

class Sampler(abc.ABC):
    # etc...
```